src/services/itemMembership/repository.ts

```diff
@@ -143,9 +145,29 @@ export const ItemMembershipRepository = AppDataSource.getRepository(ItemMembersh
         return 'item.path =' + subQuery.getQuery();
       });
 
-    if (name) {
-      query.andWhere("LOWER(item.name) LIKE '%' || :name || '%'", {
-        name: name.toLowerCase().trim(),
+    // todo: remove name
```

Comment:
You already have deprecated name, I don't think it is necessary to add a todo here. It'll be naturally deleted.
So we don't get drowned by too much todo.

With this comment, we're risking someone to actually remove name here, without removing the whole deprecated feature.